tests/test_x_nwb.py

```diff
@@ -26,6 +26,10 @@
 pytestmark = pytest.mark.xnwbtest
 
 
+pytest.skip("All x-nwb tests fail",
+            allow_module_level=True)
```

Comment:
What is the plan when the test is fixed? Will there still be a need to selectively skip the test (and fixture parameterizations?)

Reply:
my hope that Thomas Braun who will be fixing those test will also help to figure out how to avoid loading test data if tests are skipped.